misc/userscripts/cast

```diff
@@ -133,23 +144,41 @@ echo "jseval -q $(printjs)" >> "$QUTE_FIFO"
 
 tmpdir=$(mktemp -d)
 file_to_cast=${tmpdir}/qutecast
-program_=$(command -v castnow)
+cast_program=$(command -v castnow)
+
+# load optional config
+QUTE_CONFIG_DIR=${QUTE_CONFIG_DIR:-${XDG_CONFIG_HOME:-$HOME/.config}/qutebrowser/}
+QUTE_CAST_CONFIG=${QUTE_CAST_CONFIG:-${QUTE_CONFIG_DIR}/cast_rc}
+if [ -f "$QUTE_CAST_CONFIG" ] ; then
+    # shellcheck source=/dev/null
+    source "$QUTE_CAST_CONFIG"
+fi
 
-if [[ "${program_}" == "" ]]; then
+# if ytdl_program wasn't specified in config, use a fallback
+for p in "$ytdl_program" yt-dlp youtube-dl; do
+  ytdl_program=$(command -v "$p")
+  [ "$ytdl_program" == "" ] || break
+done
+
+if [[ "${cast_program}" == "" ]]; then
     msg error "castnow can't be found..."
     exit 1
 fi
+if [[ "${ytdl_program}" == "" ]]; then
+    msg error "youtube-dl (or a drop-in replacement) can't be found..."
+    exit 1
+fi
 
 # kill any running instance of castnow
-pkill -f "${program_}"
+pkill -f "${cast_program}"
 
 # start youtube download in stream mode (-o -) into temporary file
-youtube-dl -qo - "$1" > "${file_to_cast}" &
+${ytdl_program} -qo - "$1" > "${file_to_cast}" &
```

Comment:
Since `$ytdl_program`, at this point, is the result of `$(command -v "$p")` (see [line159](https://github.com/qutebrowser/qutebrowser/pull/7074/files#diff-34b4768e7037f827d7c3cf3006f417c522b580730cdc1e94506f202c87f7ac83R159)), this expansion should be quoted and it is incorrect to not quote it.

```suggestion
"${ytdl_program}" -qo - "$1" > "${file_to_cast}" &
```